lib/rbac/filterer.rb

```diff
@@ -175,9 +175,6 @@ def search(options = {})
         ids_clause = ["#{klass.table_name}.id IN (?)", target_ids] if klass.respond_to?(:table_name)
       else # targets is a scope, class, or AASM class (VimPerformanceDaily in particular)
         targets = to_class(targets)
-        # could just call all on everything, but that will display deprecation warnings
-        targets = targets.all if targets < ActiveRecord::Base || targets.kind_of?(ActsAsArModel)
-
         scope = apply_scope(targets, scope)
```

Comment:
I think the naming of targets is throwing me off, because when I think targets I'm thinking an Array (or scope) of objects.  But the to_class call has turned that into a class, so perhaps `targets_class` is a better name for clarity?  Not necessary for this PR, just trying to understand the code because it's confusing.